Scoop/Controller/MatchesViewController.swift

```diff
@@ -43,22 +78,210 @@ class MatchesViewController: UIViewController {
         view.addSubview(tableView)
         
         tableView.snp.makeConstraints { make in
-            make.top.equalTo(view.safeAreaLayoutGuide)
-            make.leading.trailing.equalToSuperview()
+            make.top.equalTo(filterView.snp.bottom).inset(-10)
+            make.leading.trailing.equalToSuperview().inset(20)
             make.bottom.equalTo(view.safeAreaLayoutGuide)
         }
     }
+    
+    private func setupTripDetails() {
+        let iconTextSpacing = 10
+        let iconSize = 20
+        let spacing = 16
+        
+        tripDetailsView.backgroundColor = .white
+        tripDetailsView.addDropShadow()
+        tripDetailsView.layer.cornerRadius = 10
+        tripDetailsView.addSubview(departureIconImageView)
+        
+        departureIconImageView.snp.makeConstraints { make in
+            make.size.equalTo(iconSize)
+            make.leading.equalToSuperview().inset(spacing)
+            make.top.equalToSuperview().inset(spacing)
+        }
+        
+        departureLocationLabel.text = departureName
+        departureLocationLabel.font = .systemFont(ofSize: 16)
+        tripDetailsView.addSubview(departureLocationLabel)
+        
+        departureLocationLabel.snp.makeConstraints { make in
+            make.leading.equalTo(departureIconImageView.snp.trailing).inset(-iconTextSpacing)
+            make.trailing.equalToSuperview().inset(spacing)
+            make.top.equalTo(departureIconImageView)
+        }
+        
+        tripDetailsView.addSubview(iconSeparatorImageView)
+        
+        iconSeparatorImageView.snp.makeConstraints { make in
+            make.height.equalTo(8)
+            make.width.equalTo(1.5)
+            make.leading.equalTo(departureIconImageView).inset(8)
+            make.top.equalTo(departureIconImageView.snp.bottom).inset(-5)
```

Comment:
try to use offset instead of inset if you aren't setting relative to the superview